subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -381,6 +381,18 @@ config BT_CTLR_TX_RETRY_DISABLE
 	  would happen in the next connection event instead of repeated retries
 	  in the current connection event.
 
+config BT_CTLR_FORCE_MD_COUNT
+	int "Forced MD bit count"
+	range 0 255
+	default 0
+	help
+	  No. of times to force MD bit to be set in Tx PDU after a successful
+	  transmission of non-empty PDU.
+
+	  This will prolong the connection event to from being closed in cases
```

Comment:
Would it make sense to allow runtime reconfiguration of this value? When it's non zero does it mean higher power consumption for all connections?

Reply:
> Would it make sense to allow runtime reconfiguration of this value?

I cant think of a easy way for this value to be determined at runtime.
If the rate of Tx buffer supplied is 2.6 ms, and a BLE Trx for empty PDU takes 388us, this number shall be higher than ~7.

> When it's non zero does it mean higher power consumption for all connections?

Yes, power consumption per connection event with non-empty PDU will be higher. I will make this feature kick in if Trx count in a radio event is BT_CTLR_TX_BUFFERS or more, this way only high throughput scenario will be affected.